admin/tools/docker/deployment/in2p3/env.example.sh

```diff
@@ -0,0 +1,20 @@
+
+# Set nodes names
+MASTER=ccqserv100.in2p3.fr
+WORKERS=$(echo ccqserv1{01..24}.in2p3.fr)
+#MASTER=ccqserv125.in2p3.fr
+#WORKERS=$(echo ccqserv1{26..49}.in2p3.fr)
+
+# set image names
```

Comment:
:+1:  parallel/env.example.sh was obsolete, replaced it with in2p3/env.example.sh